Gems/ROS2/Code/Include/ROS2/Lidar/LidarRaycasterBus.h

```diff
@@ -166,6 +168,13 @@ namespace ROS2
             AZ_Assert(false, "This Lidar Implementation does not support entity exclusion!");
         }
 
+        //! Configures segmentation classes.
+        //! @param classTags Set of pairs of class names and their corresponding class indices.
+        virtual void ConfigureSegmentationClasses(
+            [[maybe_unused]] const AZStd::set<AZStd::pair<AZStd::string, uint8_t> >& classTags) {
```

Comment:
Formatting here seems still off - is this after running clang-format? If yes, can you correct this manually?
+
Lines 100-101 above seem to have the same issue. This may also refer to other places in this PR.